### PR TITLE
[pipeline] Use batch_size in runtime_state instead of vector_chunk_size

### PR DESCRIPTION
**Pull request by Pslydhh**
- Use batch_size in runtime_state instead of vector_chunk_size

### CODE REVIEW

File: be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp [hunk 1]
```diff
@@ -60,7 +60,7 @@ Status AggregateBlockingSinkOperator::push_chunk(RuntimeState* state, const vect
              _aggregator->conjunct_ctxs().empty() &&       // no 'having' clause
              _aggregator->get_aggr_phase() == AggrPhase2); // phase 2, keep it to make things safe
     const auto chunk_size = chunk->num_rows();
-    DCHECK_LE(chunk_size, config::vector_chunk_size);
+    DCHECK_LE(chunk_size, state->query_options().batch_size);
```
Comment (trueeyu) [inline be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp:63]: why not directy use state->batch_size();

Comment (Pslydhh) [reply be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp]: Have been fixed

**Comment by trueeyu** [comment]
Current the default batch is 1024, should it be changed to 4096 ?

**Comment by trueeyu** [comment]
ColumnPool will delete the column (which size > config::vector_chunk_size), 
Does this logic affect your changes?

**Comment by Pslydhh** [comment]
> ColumnPool will delete the column (which size > config::vector_chunk_size), Does this logic affect your changes?



> Current the default batch is 1024, should it be changed to 4096 ?

**Comment by Pslydhh** [comment]
run starrocks_be_unittest

**Comment by Pslydhh** [comment]
> config::vector_chunk_size

Use batch 

> ColumnPool will delete the column (which size > config::vector_chunk_size), Does this logic affect your changes?

Use batch_size instead of config::vector_chunk_size, maybe colume's size changed in runtime, but only column from ColumnPool will be affect by this change, so It affect only column instances saved in ColumnPool.

**Comment by Pslydhh** [comment]
> Current the default batch is 1024, should it be changed to 4096 ?

Yes, its' value will be default 4096 for non-pipeline and 16384 for pipeline. It only be used in ExecNode/EsHttpScanNode/SelectOperator now.

**Comment by Pslydhh** [comment]
run all

**Comment by Pslydhh** [comment]
run starrocks_fe_unittest

**Review by kangkaisen — APPROVED** [review]

**Review by satanson — APPROVED** [review]

**Pull request merged**